### PR TITLE
Interest per block value and amounts on getvault

### PR DESCRIPTION
**Pull request by hidiego**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Add interest per block amounts for `getvault` and total value of interests per block in usd.

Tests added.

### CODE REVIEW

File: src/masternodes/rpc_vault.cpp [hunk 1]
```diff
@@ -179,6 +187,10 @@ namespace {
             interestValue = ValueFromAmount(totalInterests);
             loanBalances = AmountsToJSON(totalBalances);
             interestAmounts = AmountsToJSON(interestBalances);
+            if(verbose) {
```
Comment (prasannavl) [inline src/masternodes/rpc_vault.cpp:190]:
```suggestion
            if (verbose) {
```

:)

File: src/masternodes/rpc_vault.cpp [hunk 2]
```diff
@@ -207,6 +220,8 @@ namespace {
                 nextCollateralRatio = int(rate.val->ratio());
                 result.pushKV("nextCollateralRatio", nextCollateralRatio);
             }
+            result.pushKV("totalInterestPerBlockValue", totalInterestsPerBlockValue);
```
[the remark this answers is not in the document]
Comment (hidiego) [reply src/masternodes/rpc_vault.cpp:223]: We are using Value suffix for all other transformations, should we make an exception with this one?

[the remark this answers is not in the document]
Comment (prasannavl) [reply src/masternodes/rpc_vault.cpp:223]: Apologies, my mistake. You are correct. Overlooked that it was value.

[the remark this answers is not in the document]
Comment (prasannavl) [reply src/masternodes/rpc_vault.cpp:224]: How about `interestPerBlockValue` and then for the next one `interestsPerBlock`?

**Pull request merged**